.github/workflows/quality_check.yml

```diff
@@ -37,6 +37,8 @@ jobs:
         run: pip3 install -r requirements.txt 
 
       - name: Run tests
+        env : 
+          SCHEMA_NAME : ${{ secrets.SCHEMA_NAME }}
```

Comment:
Added SCHEMA_NAME to repo secrets to get tests relying on env variables, passing